.github/workflows/php-compat.yml

```diff
@@ -0,0 +1,45 @@
+name: PHP Compatibility
+
+on:
+  push:
+    branches:
+      - develop
+      - trunk
+    paths:
+      - "**.php"
+  pull_request:
+    branches:
+      - develop
+    paths:
+      - "**.php"
```

Comment:
Could you please add the PR branch `add/linting-php-compat-actions` to this list? This way, we can verify if the actions run and pass as expected.